pkgs/development/libraries/libjodycode/default.nix

```diff
@@ -32,8 +35,8 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Shared code used by several utilities written by Jody Bruchon";
     homepage = "https://github.com/jbruchon/libjodycode";
-    changelog = "https://github.com/jbruchon/libjodycode/blob/${src.rev}/CHANGES.txt";
+    changelog = "https://github.com/jbruchon/libjodycode/blob/${finalAttrs.src.rev}/CHANGES.txt";
```

Comment:
This github repository is nonexistant.

Also https://github.com/NixOS/nixpkgs/issues/336454

Reply:
https://codeberg.org/jbruchon/libjodycode/src/branch/master/CHANGES.txt

Reply:
This link points to github, not codeberg.